src/main/scala/org/camunda/package.scala

```diff
@@ -120,13 +120,13 @@ package object feel {
     localDateTimePatter.matcher(dateTime).matches
 
   private lazy val yearMonthDurationPattern =
-    Pattern.compile("""-?P(\d+Y)?(\d+M)?""")
+    Pattern.compile("""-?P(\d+Y)?-?(\d+M)?""")
 
   def isYearMonthDuration(duration: String): Boolean =
     yearMonthDurationPattern.matcher(duration).matches
 
   private lazy val dayTimeDurationPattern =
-    Pattern.compile("""-?P((\d+D)?(T(\d+H)?(\d+M)?(\d+(.\d*)?S)?)?)""")
+    Pattern.compile("""-?P((\d+D)?(T(\d+H)?-?(\d+M)?(\d+(.\d*)?S)?)?)""")
```

Comment:
:x: We need to adjust the pattern a bit more.

The minus sign could be placed on each component. Currently, it doesn't work for:
- `P-1D`
- `PT-2H`
- `PT-3M-4S`